src/test/java/org/jenkinsci/plugins/envinject/BuildCauseRetrieverTest.java

```diff
@@ -1,121 +0,0 @@
-package org.jenkinsci.plugins.envinject;
```

Comment:
In such case you should manually notify git about moves or split move and further changes into two commits